src/daemon/ca/manager.rs

```diff
@@ -1071,32 +1035,20 @@ impl CaManager {
     ) -> KrillResult<api::Entitlements> {
         let ca = self.get_ca(&handle).await?;
         let contact = ca.parent(parent)?;
-        self.get_entitlements_from_parent_and_contact(handle, parent, contact)
-            .await
+        self.get_entitlements_from_parent_and_contact(handle, contact).await
     }
 
     pub async fn get_entitlements_from_parent_and_contact(
```

Comment:
Since parent is no longer passed in, remove the word parent from the fn name?

Reply:
ack